packages/one-app-runner/src/startApp.js

```diff
@@ -131,15 +132,10 @@ const generateDebug = (port, useDebug) => (useDebug ? `--inspect=0.0.0.0:${port}
 // So we have to remove those flags if the one-app version is less than 5.13.0
 // 5.13.0 is when node 16 was introduced.
 const generateNodeFlags = (appVersion) => {
-  const versionArray = appVersion.split('.');
-  if (versionArray.length > 1) {
-    const majorVersion = Number.parseInt(versionArray[0], 10);
-    const minorVersion = Number.parseInt(versionArray[1], 10);
-    if (majorVersion === 5 && minorVersion < 13) {
-      return '';
-    }
+  if (semver.intersects(appVersion, '^5.13.0', { includePrerelease: true })) {
```

Comment:
i didn't know that this existed until now, and thanks to your refactor it will be more readable. Could we maybe extract this information and make it a constant? I feel like we won't never see it until we actually read this piece of code, couple of ideas:

1. Access the node version, like `process.env`
2. Make it a constant, something like the following, so we can at least know what `5.13.0` means without the need of comments:
```js
const ONE_APP_MIN_NODE_VERSION_MAP = {
   '^5.13.0': 16
}
```
3. Just add `NOTE:` so we can programmatically find this